test/e2e/tests/evidence_test.go

```diff
@@ -53,3 +55,13 @@ func TestEvidence_DoubleVote(t *testing.T) {
 
 	})
 }
+
+func fetchMisebehaviors(t *testing.T) {
+	t.Helper()
+
+	misbehavior = 
```

Comment:
?

Reply:
Sorry this is still a wip. I'm just committing stuff here so I can transfer it to my laptop